Web/starter_project_vanguard/styles/globals.css

```diff
@@ -17,11 +18,11 @@
 }
 
 body {
-  color: rgb(var(--foreground-rgb));
+  /* color: rgb(var(--foreground-rgb));
```

Comment:
better not to leave in comments in the final code
if you want to remove it then remove it 
we can always recover the code from the git history